e2e/tests/bulkScanUserUploadsScannedDocuments_test.js

```diff
@@ -0,0 +1,49 @@
+const config = require('../config.js');
+const scannedDocument = require('../fixtures/scannedDocument.js');
+
+let caseId;
+
+Feature('Uploading bulk scan document');
+
+Before(async (I, caseViewPage, submitApplicationEventPage) => {
+
+  if (!caseId) {
+    await I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
+    await I.enterMandatoryFields({multipleChildren: false});
+    await caseViewPage.goToNewActions(config.applicationActions.submitCase);
+    submitApplicationEventPage.giveConsent();
+    await I.completeEvent('Submit');
+
+    // eslint-disable-next-line require-atomic-updates
+    caseId = await I.grabTextFrom('.heading-h1');
+    console.log(`Case ${caseId} has been submitted`);
+  }
+  I.signOut();
+});
+
+Scenario('Bulk scan user uploads Scanned Documents', async (I, caseViewPage, handleSupplementaryEvidenceEventPage, attachScannedDocsEventPage) => {
```

Comment:
personally i would not make this a scenario, as it it not real business use case, it can be just data preparation step for 'HMCTS admin can see Documents scanned in with Bulk Scan'

Reply:
hmm, whilst I agree it doesn't add buisness value, it is the only test we have to assert that the event works as expected 🤔